src/globals/scss/_helper-mixins.scss

```diff
@@ -183,6 +183,31 @@
   }
 }
 
+@mixin track($fill: 0) {
```

Comment:
What is this function about?

Reply:
Was testing out something, forgot to remove it. Thanks for the heads up